internal/sbom/spdx.go

```diff
@@ -40,82 +38,186 @@ func GenerateSPDX(koVersion string, date time.Time, mod []byte, imgDigest v1.Has
 		return nil, err
 	}
 
+	doc := Document{
+		Version:     "SPDX-2.2",
+		DataLicense: "CC0-1.0",
+		ID:          "SPDXRef-DOCUMENT",
+		Name:        bi.Main.Path,
+		Namespace:   "http://spdx.org/spdxpackages/" + bi.Main.Path,
```

Comment:
/spdxdocs is the public url for namespacing the SBOMs ([see here](https://spdx.github.io/spdx-spec/document-creation-information/#65-spdx-document-namespace-field))
```suggestion
		Namespace:   "http://spdx.org/spdxdocs/" + bi.Main.Path,
```

Reply:
done!  this was carryover from what we did before.